src/azbridge/azbridge.csproj

```diff
@@ -135,15 +135,6 @@
     <RpmDependency Include="zlib" Version="" />
   </ItemGroup>
 
-  <ItemGroup Condition="$(RuntimeIdentifier.StartsWith('debian')) OR $(RuntimeIdentifier.StartsWith('ubuntu')) OR $(RuntimeIdentifier.StartsWith('linuxmint'))">
-    <!-- <DebDependency Include="lttng-ust" Version=""/> -->
-    <DebDependency Include="libcurl3 | libcurl4" Version=""/>
```

Comment:
Should I remove all the `<RpmDependency>` and `<RpmDotNetDependency>` elements too?  Do I need to manually test that, or will the PR/CI builds do that automatically?